### PR TITLE
upload Windows startup scripts to GCS for CI

### PR DESCRIPTION
**Pull request by mtaufen**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Upload the Windows startup scripts to where GKE can see them, similar to COS/Ubuntu scripts.

```release-note
NONE
```

@yujuhong @pjh FYI
@ixdy Does this look right to you?

### CODE REVIEW

**Comment by ixdy** [comment]
This looks technically correct, but I'll echo the same comment I made over on https://github.com/kubernetes/release/pull/662 - why is Google/GKE/GCE special here?

**Comment by mtaufen** [comment]
As soon as we can get rid of the deprecated `cluster/` directory we can get rid of this too, but until then we should maintain the legacy behavior.

**Comment by mtaufen** [comment]
/hold
this on the switch to 1809 same as https://github.com/kubernetes/release/pull/662#pullrequestreview-202435127

File: cluster/gce/win1803/BUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Having the Windows code from the GCE cluster deploy hosted with the release is
+# useful for GKE.  This list should match the list in
+# kubernetes/release/lib/releaselib.sh.
+release_filegroup(
+    name = "gcs-release-artifacts",
+    srcs = [
+        "common.psm1",
+        "configure.ps1",
+        "k8s-node-setup.psm1",
+        "user-profile.psm1",
```
Comment (pjh) [inline cluster/gce/win1803/BUILD]: I'm updating this directory structure (and adding a file) in https://github.com/kubernetes/kubernetes/pull/74360.

**Comment by pjh** [comment]
Our PR to switch the test job to 1809 has been merged ([kubernetes/kubernetes#74103](https://github.com/kubernetes/kubernetes/pull/74103)). However, I suggest waiting for [kubernetes/kubernetes#74360](https://github.com/kubernetes/kubernetes/pull/74360) to be merged and then updating this PR to reflect the files correctly - I added a testonly/ subdir and moved a couple files there.

**Comment by mtaufen** [comment]
/hold cancel
Updated for latest upstream

File: cluster/gce/windows/BUILD [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# We HIGHLY recommend not changing the file structure, because consumers of
+# Kubernetes releases depend on the release structure remaining stable.
+release_filegroup(
+    name = "gcs-release-artifacts",
+    srcs = [
+        "common.psm1",
+        "configure.ps1",
+        "k8s-node-setup.psm1",
+        "testonly/install-ssh.psm1",
+        "testonly/user-profile.psm1",
```
Comment (ixdy) [inline cluster/gce/windows/BUILD]: in https://github.com/kubernetes/release/pull/662/files this doesn't have a `testonly/` prefix?

File: cluster/gce/windows/BUILD [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# and `k8s.io/release/lib/releaselib.sh` must be manually updated with the changes.
+# We HIGHLY recommend not changing the file structure, because consumers of
+# Kubernetes releases depend on the release structure remaining stable.
+release_filegroup(
+    name = "gcs-release-artifacts",
+    srcs = [
+        "common.psm1",
+        "configure.ps1",
+        "k8s-node-setup.psm1",
+        "testonly/install-ssh.psm1",
```
Comment (ixdy) [inline cluster/gce/windows/BUILD]: this isn't included in https://github.com/kubernetes/release/pull/662?

Comment (mtaufen) [reply cluster/gce/windows/BUILD]: Thanks! Updated that PR to include it.

File: cluster/gce/windows/BUILD (hunk 2 above)
Comment (mtaufen) [reply cluster/gce/windows/BUILD]: Thanks! Updated that PR as well.

**Comment by ixdy** [comment]
/lgtm
/approve

I'm not especially a fan of adding more GKE-specific bits to the GCS path, but I guess the plan is to eventually retire `cluster/`.

**Pull request merged**